### PR TITLE
Replace 'sequelize.sync()' with 'yarn migrate' 

### PR DESCRIPTION
**Pull request by smallbee3**
This is the pull request in regard with the issue #48 
So, I recommend you to look at the issue first and read the code.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
   "author": "dooboolab",
   "license": "GPL-3.0-only",
   "scripts": {
-    "local": "cross-env NODE_ENV=local nodemon --exec ts-node src/server.ts",
-    "start": "cross-env NODE_ENV=development nodemon --exec ts-node src/server.ts",
+    "local": "cross-env NODE_ENV=local sequelize db:migrate && cross-env NODE_ENV=local nodemon --exec ts-node src/server.ts",
```
Comment (hyochan) [inline package.json]: You don't need `cross-env` twice here.

File: package.json [hunk 2]
```diff
@@ -7,8 +7,8 @@
...
+    "start": "cross-env NODE_ENV=development sequelize db:migrate && cross-env NODE_ENV=development nodemon --exec ts-node src/server.ts",
```
Comment (hyochan) [inline package.json]: Same here!

File: src/db/index.ts [hunk 3]
```diff
@@ -14,8 +14,4 @@ const sequelize = new Sequelize(
   },
 );
 
-if (process.env.NODE_ENV !== 'test') {
-  sequelize.sync();
-}
```
Comment (hyochan) [inline src/db/index.ts]: Could you tell why you've removed this?

Comment (hyochan) [reply src/db/index.ts]: Oh, I read your #48 now. Understood!

**Comment by smallbee3** [comment]
I updated the pull request because I missed the model associations part.
I may need to do some api tests !

File: migrations/20200215075119-Membership.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+          type: UUID,
+          references: {
+            model: 'channels',
+            key: 'id'
+          },
+          allowNull: true,
+        },
+      },
+      {
+        charset: 'utf8',
```
Comment (hyochan) [inline migrations/20200215075119-Membership.js]: utf8mb4

**Comment by hyochan** [comment]
@smallbee3 Please tell me when this is ready! (Since you told me that this is pretty urgent)

**Comment by smallbee3** [comment]
Sorry for the late update.

In the latest commits, I've done some work as below.

1. Add model relations(assocations)
2. Change charset option from 'utf8' to 'utf8mb4'
3. Remove 'paranoid' option and 'deletedAt' field from 'messages' model
4. Add "*.sql" to gitignore

Thanks

**Comment by smallbee3** [comment]
Sorry for the late update.

In the latest commits, I've done some work as below.

1. Add model relations(assocations)
2. Change charset option from 'utf8' to 'utf8mb4'
3. Remove 'paranoid' option and 'deletedAt' field from 'messages' model
4. Add "*.sql" to gitignore

I force pushed after the rebase to the upstream master.

Thanks

**Pull request merged**